homeassistant/components/mqtt/server.py

```diff
@@ -85,6 +98,6 @@ def generate_config(hass, passwd):
         username = None
         password = None
 
-    client_config = ('localhost', 1883, username, password, None, PROTOCOL_311)
+    client_config = ('localhost', 1883, username, password, None, '3.1.1')
```

Comment:
You should not import from hbmqtt because it is the protocol spec for the paho mqtt client.